src/rtr/server.rs

```diff
@@ -25,7 +25,11 @@ use super::state::State;
 /// The maximum protocol version we support.
 ///
 /// We support all protocol versions from 0 up to and including this value.
-const MAX_VERSION: u8 = 2;
+///
+/// While the server technically supports version 2 as well, the format of the
+/// ASPA PDU has not yet been agreed upon. Rather than possibly deploying
+/// broken servers, we only announce support for version 0 or 1 for now.
+const MAX_VERSION: u8 = 1;
```

Comment:
It's perhaps unnecessary but to avoid re-enabling this / proving that it has the desired effect, should there be a unit test showing that version 2 is rejected?

Reply:
There should be. But we currently don’t have any scaffolding for these tests at all. Which is to say: The entire RTR implementation needs tests.